docs/workflows/test-clever-users.md

```diff
@@ -0,0 +1,30 @@
+# How to Impersonate a Clever User on Localhost
+
+This guide walks you through the steps needed to impersonate a Clever user on your localhost for testing purposes.
+
+## Steps to Impersonate a Clever User
+
+1. **Start the Application in Development Mode**
+   - Run the following command on your terminal:
+     ```bash
+     npm run dev:live
+     ```
+   This will start the development server and prepare the app for Clever testing.
```

Comment:
Could we elaborate why we need to start the application with `dev:live` and not the regular `dev`?

Reply:
yes!